core/trino-main/src/main/java/io/trino/operator/join/PageJoiner.java

```diff
@@ -59,11 +59,11 @@
 import static java.util.Objects.requireNonNull;
 
 public class PageJoiner
-        implements WorkProcessor.Transformation<Page, Page>
+        implements PageJoinerTransformation
```

Comment:
I would rather have interface named `PageJoiner` and rename implementation to e.g. `DefaultPageJoiner`